### DISCUSSION TITLE
Red Pitaya "Hall of fame" blog post + version of OS?

### DISCUSSION THREAD

**Discussion by Lightsaver7** [linien-org/linien#362] [General]
Dear B. Wiegand, B. Leykauf, R. Jördens, and M. Krutzik,

Thank you for creating such a cool application for the Red Pitaya!

Last week, I accidentally found your project on GitHub while searching for Laser Lock-In options for Red Pitaya. After reading the project description and seeing that it is still in development, I realised it is prime material for a blog post on Red Pitaya's website (Hall of Fame) so in the name of the Red Pitaya team, I am asking for your permission to do so. Here is the link to the blog: https://content.redpitaya.com/blog

Besides this, I have one question regarding the Red Pitaya OS version this is meant to operate on. Was the project made to be compatible with the Red Pitaya 2.00 OS (I assume it is, due to uploading the bitstream file as .bin)? Nowhere in the documentation did I find any mention of this.

Good luck with your project!

If you have any questions, you can contact me through my email below or write to support@redpitaya.com.

Sincerely,

Miha Gjura
Technical Specialist

E: miha.gjura@redpitaya.com
W: www.redpitaya.com

Red Pitaya d.o.o
Velika pot 21
5250 Solkan, Slovenija (EU)

**Comment by bleykauf** [comment]
I would be honored if Linien is featured on the RedPitaya Blog :) (I guess @hermitdemschoenenleben and @jordens also don't mind)

However, there are a small number of bugs that I would like to still fix by the end of the year, specifically https://github.com/linien-org/linien/issues/360. I would like to then tag this as version 1.0 which would mark a good point for the blog post. I would then come back to you, @Lightsaver7. I think we can then talk about details via email.

Regarding compatability: Linien was originally developed vor RedPitaya OS 1.x but since version v.0.8.0, it is also compatible with OS 2.x. The deployment procedure can be found [here](https://github.com/linien-org/linien/blob/47f8c58477ff30f07147fbc98f251446365a95e3/linien-server/linien_server/acquisition.py#L308-L322). Since the Python version on OS 1.x is very old, I plan to remove support for OS 1.x following the release of Linien 1.0 (except maybe some smaller bug fixes if they pop up).

**Comment by Lightsaver7** [comment]
That is great news! Thank you.

Please, take your time to fix the bugs, and tell me when the project is ready 😃.

It would be great if you could just add a note somewhere in the documentation specifying the OS compatibility so that this is clear to anyone reading it.

**Reply by bleykauf** [reply]
It actually went much faster than anticipated and I am happy to report that I did just release version 1.0.0. From my side everything is ready for the blog post. I just wrote you an email so we can discuss details :)

RedPitaya OS compatibility is now mentioned in the Installation instructions and the release notes.

**Reply by Lightsaver7** [reply]
Thank you for including the compatibility note. That will help the users a lot 😄.

**Comment by jordens** [comment]
Fine by me!
